velox/functions/sparksql/aggregates/SumAggregate.cpp

```diff
@@ -45,22 +45,27 @@ void checkAccumulatorRowType(const TypePtr& type) {
 std::unique_ptr<exec::Aggregate> constructDecimalSumAgg(
     const TypePtr& inputType,
     const TypePtr& sumType,
-    const TypePtr& resultType) {
+    const TypePtr& resultType,
+    core::AggregationNode::Step step,
+    const std::vector<TypePtr>& argTypes) {
```

Comment:
nit: Reorder the parameter list to be `step, argTypes, resultType, inputType, sumType`.